app/src/main/java/com/example/socialapp/SharedPreferenceHelper.java

```diff
@@ -0,0 +1,21 @@
+package com.example.socialapp;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import androidx.preference.PreferenceManager;
+
+
+public class SharedPreferenceHelper{
+    static void setSharedPreferenceString(Context context, String key, String value){
```

Comment:
we can have shorter name like `setString`

Reply:
Method should be public.

Reply:
Implemented